themes/godotengine/layouts/home.htm

```diff
@@ -329,10 +329,9 @@ <h2>A different way to make games</h2>
           loading="lazy"
         >
         <div class="body">
-          <h4>Innovative design</h4>
+          <h4>{{ TR('Innovative design') }}</h4>
           <p>
-            Godot's Node and Scene system gives you both power and flexibility
-            to create anything.
+            {{ TR('Godot’s Node and Scene system gives you both power and flexibility to create anything.') }}
```

Comment:
I'm not sure it's a good idea to use the typographical quote here. It's pretty difficult to type for most normal users and thus won't be easy to reproduce in translations, nor in new strings we write.

Reply:
I can replace them with regular ones, I like them better visually but not a big deal if we use the easier ones to type 😅

Reply:
There is something to be said about using proper punctuation for some languages. Like we use «» when we want to be "proper" and fancy with quotation marks, but I think it requires complete commitment, or it will end up being inconsistent.

Reply:
In my experience, this character is always used inconsistently because not everyone cares / knows how to type it. It's usually less work to remove the few added by mistake (typically from people who copy pasted content from MS Word which may auto-insert them in some locales) than the other way around.